packages/flutter_hooks/test/memoized_test.dart

```diff
@@ -16,7 +16,7 @@ void main() {
 
     await tester.pumpWidget(
       HookBuilder(builder: (context) {
-        ref = useRef();
+        ref = useRef<int?>(null);
```

Comment:
These tests should be unchanged. Could you instead add a new test that checks both a null and non-null default value?

Reply:
And that makes sure that when the widget rebuilds, the state is not reset

Reply:
Hope [this](https://github.com/rrousselGit/flutter_hooks/pull/244/commits/3ee484220696492a320762ebf5fdb59de18b8897) will suffice